operator/manager/teardown.go

```diff
@@ -167,6 +168,19 @@ func (t *teardown) do(ctx context.Context, stage *core.Stage) error {
 			Warnln("manager: cannot publish build event")
 	}
 
+	if t.Webhook != nil {
```

Comment:
this can be removed because `t.Webhook` is never nil.  If no webhook is defined the `Send` method is a no-op.

Reply:
Sure. I wasn't too sure so I took the cautious route just in case to prevent a nil pointer deref